packages/x-data-grid-premium/src/constants/dataGridPremiumDefaultSlotsComponents.ts

```diff
@@ -2,9 +2,11 @@ import { DATA_GRID_PRO_DEFAULT_SLOTS_COMPONENTS } from '@mui/x-data-grid-pro/int
 import type { GridPremiumSlotsComponent } from '../models';
 import { GridPremiumColumnMenu } from '../components/GridPremiumColumnMenu';
 import materialSlots from '../material';
+import { GridPinnedRows } from '../components/GridPinnedRows';
 
 export const DATA_GRID_PREMIUM_DEFAULT_SLOTS_COMPONENTS: GridPremiumSlotsComponent = {
   ...DATA_GRID_PRO_DEFAULT_SLOTS_COMPONENTS,
   ...materialSlots,
   columnMenu: GridPremiumColumnMenu,
+  pinnedRows: GridPinnedRows,
```

Comment:
Out of interest, what's the benefit to this being a slot?

Reply:
Just not to spill the logic related to the Premium package in Pro or Community.